pkg/test-infra/tidb/operation.go

```diff
@@ -459,6 +468,37 @@ func getTidbDiscoveryService(tc *v1alpha1.TidbCluster) *corev1.Service {
 	}
 }
 
+func parseConfig(config string) (string, error) {
```

Comment:
How about checking scheme first and then fallback to old behaviour?
I think we need not to check if the config file exists here. It is checked in `readFileAsString`.
Think about the case when the config value is empty.

Reply:
You are right.

Reply:
I agree with you.